CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/RemoveBlazePlugin.java

```diff
@@ -45,6 +47,7 @@
  */
 @ServiceProvider(service = Plugin.class)
 @NbBundle.Messages("RemoveBlazePlugin=Remove Blazes")
+@PluginInfo(pluginType = PluginType.DELETE, tags = {"MODIFY"})
```

Comment:
what about `DELETE`? `MODIFY` also works but was curious to the thinking behind using `MODIFY`

Reply:
I think of delete as removing elements from the graph, not specifically attributes. I can change it to delete to be more consistent?